contrastive_vi/module/contrastive_vi.py

```diff
@@ -578,60 +551,37 @@ def loss(
             inference_outputs["target"],
             generative_outputs["target"],
         )
-        recon_loss = background_losses["recon_loss"] + target_losses["recon_loss"]
-        kl_z = background_losses["kl_z"] + target_losses["kl_z"]
-        kl_s = target_losses["kl_s"]
-        kl_library = background_losses["kl_library"] + target_losses["kl_library"]
-
-        loss = (
-            torch.sum(recon_loss)
-            + torch.sum(kl_z)
-            + torch.sum(kl_s)
-            + torch.sum(kl_library)
+        reconst_loss = background_losses["recon_loss"] + target_losses["recon_loss"]
+        kl_divergence_z = background_losses["kl_z"] + target_losses["kl_z"]
+        kl_divergence_s = target_losses["kl_s"]
+        kl_divergence_l = background_losses["kl_library"] + target_losses["kl_library"]
+
+        wasserstein_loss = (
```

Comment:
Just to make sure. This is the squared Wasserstein's distance, right?

Reply:
Yes, that's correct!